.circleci/config.yml

```diff
@@ -67,11 +67,26 @@ commands:
             - /home/circleci/.composer/cache/
       - store_artifacts:
           path: ./artifacts
-      - slack/status:
-          fail_only: true
-          failure_message: ':red_circle: M2: A $CIRCLE_JOB job has failed!'
-          only_for_branches: master
-          webhook: $SLACK_GREENKEEPER_WEBHOOK
+      - run:
+          name: "Conditionally Notify Greenkeeper"
+          when: on_fail
+          command: |
+            if [[ ${CIRCLE_BRANCH} == "master" ]]
+              export NOTIFY_FLAG=true
+            else
+              if [[ ${CIRCLE_BRANCH:0:3} == ci/ ]] && grep -n "^${CIRCLE_BRANCH:3}$" ./auto-rebase-branchess.txt >> /dev/null ; then
```

Comment:
Hi @VincentGalloroBolt , there is a typo in the filename. It should be **_./auto-rebase-branches.txt_** Did you test this? Please don't rush merging PRs without testing.